rust-escrow/stable-escrow/src/escrow.rs

```diff
@@ -469,10 +479,20 @@ impl Escrow {
         .to_string()
         .into_bytes()
     }
+
+    fn measure_account_storage_usage(&mut self) {
+        let initial_storage_usage = env::storage_usage();
+        let tmp_account_id = AccountId::new_unchecked("a".repeat(64));
+        self.deposits.insert(&tmp_account_id);
+        self.ft.accounts.insert(&tmp_account_id, &0u128);
+        self.account_storage_usage = env::storage_usage() - initial_storage_usage;
+        self.ft.accounts.remove(&tmp_account_id);
+        self.deposits.remove(&tmp_account_id);
+    }
 }
 
 near_contract_standards::impl_fungible_token_core!(Escrow, ft);
-near_contract_standards::impl_fungible_token_storage!(Escrow, ft);
+//near_contract_standards::impl_fungible_token_storage!(Escrow, ft);
```

Comment:
We are removing this because it didn't work and you implemented the logic in `storage_impl`?
Why it didn't work? 

@susguzman

Reply:
We are using a vector (`deposits`) that grows for each user.  The space that this new array occupies must be charged to the users and that is why storage management must be implemented again because the NEP141 doesn’t take in account this new vector